lib/web/apiserver.go

```diff
@@ -1249,6 +1251,36 @@ func (h *Handler) getResetPasswordToken(ctx context.Context, tokenID string) (in
 	}, nil
 }
 
+// createNodeJoinToken creates a node join token and sends it back along with its expiry.
+// A node join token is used to join a node to a cluster.
+func (h *Handler) createNodeJoinToken(w http.ResponseWriter, r *http.Request, _ httprouter.Params, ctx *SessionContext) (interface{}, error) {
+	clt, err := ctx.GetClient()
```

Comment:
lets do it similarly to this:

https://github.com/gravitational/teleport.e/blob/41e86ff43eb2cbfec752bf0c46a635ce37b11005/lib/web/users.go#L42

This way it's easier to test and it does not require an integration testing. You can create a file `jointokens.go` (similarly to what you did with users)